### DISCUSSION TITLE
bug encrypt UserDefault

### DISCUSSION THREAD

**Discussion by armanhossiny** [axmolengine/axmol#1766] [General]
Hi,

There is a problem with the data saving in the encrypt userdefault. Sometimes when the game crashes, this problem occurs. This is the fourth time it has happened. To fix the problem, the userdefault file needs to be deleted. What do you think is the reason for this problem?

![image](https://github.com/axmolengine/axmol/assets/42970068/f07c9a72-ca60-4568-bb15-c0616a7d3a7d)

**Comment by halx99** [comment]
Do you start multiple process instances of your game on windows?

**Reply by armanhossiny** [reply]
Yes, I have run several processes, but each one has its own userdefault.
